Tests/Orso.Arpa.Tests.Shared/Identity/FakeUserManager.cs

```diff
@@ -25,8 +25,8 @@ public FakeUserManager()
               Substitute.For<IUserStore<User>>(),
               Substitute.For<IOptions<IdentityOptions>>(),
               Substitute.For<IPasswordHasher<User>>(),
-              new IUserValidator<User>[0],
-              new IPasswordValidator<User>[0],
+              Array.Empty<IUserValidator<User>>(),
```

Comment:
This two lines are not changing the behavior but is a replacement for a nicer code style, right?